ui/app/selectors/confirm-transaction.js

```diff
@@ -0,0 +1,65 @@
+import { createSelector } from 'reselect'
```

Comment:
Do you prefer to have these here in a general `selectors/` directory, as opposed to in one of the `confirm-transaction` directories? (As is done, for example, with send)